src/bp-templates/bp-nouveau/js/buddypress-messages.js

```diff
@@ -2524,8 +2524,20 @@ window.bp = window.bp || {};
 							}
 						} else if ( response.id ) {
 							bp.Nouveau.Messages.displayFeedback( response.feedback, response.type );
-							bp.Nouveau.Messages.router.navigate( 'view/' + response.id + '/?refresh=1', { trigger: true } );
-							bp.Nouveau.Messages.router.navigate( 'view/' + response.id + '/', { trigger: true } );
+							if ( undefined !== response.messages_count && 0 === response.messages_count ) {
+								if ( bp.Nouveau.Messages.threads.length > 1 ) {
+									bp.Nouveau.Messages.threads.remove( response.id );
+									bp.Nouveau.Messages.router.navigate( 'view/' + bp.Nouveau.Messages.threads.at( 0 ).id + '/', { trigger: true } );
+								} else {
+									BP_Nouveau.messages.hasThreads = false;
+									bp.Nouveau.Messages.threads.remove( response.id );
+									bp.Nouveau.Messages.router.navigate( 'view/?refresh=1', { trigger: true } );
+									bp.Nouveau.Messages.router.navigate( 'compose/', { trigger: true } );
+								}
```

Comment:
@nitishkaila Run the PHPCBF to make tabbing proper aligned.

Reply:
@chetansatasiya I have run the PHPCBF for this js file changes. But not found any changes.